configuration/configuration_test.go

```diff
@@ -8,13 +8,15 @@ import (
 	"github.com/almighty/almighty-core/configuration"
 	"github.com/almighty/almighty-core/resource"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 const (
 	tDefaultConfigurationFile           = "config.yaml"
 	tEnvironmentVariableNameConfigFile  = "ALMIGHTY_CONFIG_FILE_PATH"
 	tEnvironmentVariableValueConfigFile = "../config.yaml"
 	tInvalidConfigFilePath              = "../invalid_config.yaml"
+	tTestConfigFilePath                 = "../test/data/multiple_config.yaml"
```

Comment:
It seems like `tTestConfigFilePath` is not used?! What does "multiple_config.yaml" mean?